features/lwipstack/lwip/src/core/ipv4/lwip_icmp.c

```diff
@@ -371,14 +371,18 @@ icmp_send_response(struct pbuf *p, u8_t type, u8_t code)
           IP_HLEN + ICMP_DEST_UNREACH_DATASIZE);
 
   ip4_addr_copy(iphdr_src, iphdr->src);
+
+  char name [INTERFACE_NAME_SIZE];
+  sprintf(name, "%c%c%d", p->netif->name[0], p->netif->name[1], p->netif->num);
```

Comment:
This is made in several places so I think there should be helper functions for this. Also the p->netif->num is running from 0 to 255 so more space should be reserved for it.